README.md

```diff
@@ -7,7 +7,7 @@ Command to delete lead_event_log table entries, campaign_lead_event_log table en
 Important: If referenced email is ever switched back to published, the contacts will get the email again.
 
 ```
-bin/console mautic:leuchtfeuer:housekeeping
+bin/console Leuchtfeuer:housekeeping
```

Comment:
Lowercase command name

Reply:
Changed to lower case with commit 3979ef0